src/views/shared/SecurityBeacon.js

```diff
@@ -28,14 +43,19 @@ define(['okta', 'util/Animations'], function (Okta, Animations) {
     el.css('background-image', '');
     el.removeClass('new-user undefined-user');
     if (isNewUser) {
+      el.find('.auth-beacon-description').text('no security image');
       el.addClass('new-user');
       return;
     }
     if (isUndefinedUser) {
+      el.find('.auth-beacon-description').text('no security image');
       el.addClass('undefined-user');
       return;
     }
     if (isSecurityImage) {
+      // TODO: Newer versions of qtip will remove aria-describedby on their own when destroy() is called.
+      el.removeAttr('aria-describedby');
```

Comment:
Yes - the screenreader doesn't pay attention to alt text for images loaded as background-image. I believe it does satisfy the standard if it is contained in the same DOM element.